protocol/x/blocktime/keeper/msg_server_test.go

```diff
@@ -61,13 +61,13 @@ func TestMsgUpdateParams(t *testing.T) {
 				Authority: authtypes.NewModuleAddress(govtypes.ModuleName).String(),
 				Params: types.DowntimeParams{
 					Durations: []time.Duration{
-						1,
+						5 * time.Second,
```

Comment:
Can you update the above test case name?

Reply:
wow, good eyes!